### ISSUE TITLE
calling cleanUp() on an overlay that has disableClick = true

### ISSUE THREAD

**Issue by tomafc330** [worker8/TourGuide#39]
Hi there,
I seem to be having an issue with dismissing the overlay, this is my init code:

```
      mTourGuideHandler = TourGuide.init(this).with(TourGuide.Technique.Click)
          .setPointer(new Pointer())
          .setToolTip(null)
          .setOverlay(new Overlay().disableClick(true));
```

and later on I clean it up:

```
          mTourGuideHandler.cleanUp(); // end tour
```

However the overlay does not disappear, is there a reason why?

**Comment by TheRishka** [comment]
Same thing for me. I'm trying to dismiss tutorial on overlay click, but it doesn't work. Exactly the same code as in OverlayCustomisationActivity (just copy-pasted) - no effect.

**Comment by tomafc330** [comment]
@TheRishka actually i figured out the issue. Make sure you are not calling it twice or putting the init code inside onResume(). The instance var only holds the last instance of the TourGuide so even when you cancel that instance, the previous instance is still shown.

**Comment by worker8** [comment]
I'm trying to understand more about this issue.

@TheRishka Do you mean that when you run OverlayCustomisationActivity directly it works fine, but when you copy and paste it into your app, it doesn't work anymore? If so, there must be some difference, can you help finding out?

@tommytcchan You seem to understand more on this issue, but I still don't quite understand from what you described. Can you explain a bit more on how I can reproduce it? (some code example will be great :D)

**Comment by TheRishka** [comment]
@TheRishka yes, there is a huge difference, which i suppose is the main reason. I'm using google maps fragments together with TourGuide. It looks like it intercepts touch events. So i had to create a phantom transparent fragment, bring it in front of gmaps fragment and set onTouchListener on it.
